### DISCUSSION TITLE
Support for rollup?

### DISCUSSION THREAD

**Discussion by michael-psenka** [jheddings/notional#188] [Q&A]
(notional v0.8.0)

Putting this here instead of an issue since I'm unsure if the issue is notional or something underlying (pydantic or my understanding of Notion API).

If my database has a rollup column, I get the following error when trying to retrieve via `db = notion.databases.retrieve("MY_DB_ID")`:

`ValidationError: 1 validation error for Database
properties -> Property -> rollup -> function
  value is not a valid enumeration member`

with the following error trace:

``` python
File [~/.local/lib/python3.8/site-packages/notional/session.py:295], in DatabasesEndpoint.retrieve(self, dbref)
    291 logger.info("Retrieving database :: %s", dbid)
    293 data = self().retrieve(dbid)
--> 295 return Database.parse_obj(data)

File [~/.local/lib/python3.8/site-packages/pydantic/main.py:526], in pydantic.main.BaseModel.parse_obj()

File [~/.local/lib/python3.8/site-packages/pydantic/main.py:341], in pydantic.main.BaseModel.__init__()
```

If I delete the rollup column from Notion (no change to code), there's no error and I can retrieve perfectly fine. My integration has access to everything that the rollup column references.

**Comment by Bullish-Design** [comment]
Following up on this. I've rewritten multiple files in my local Notional codebase to "filter" out rollup columns, but it keeps popping up every time I try something new. Right now I just want to ignore all rollup columns, and there has to be a better way to temporarily patch it than sneaking in between every call to a database and iterating through the return data, excluding every column with a rollup type. 

Thoughts/recommendations, anyone?
